tests/ensure_provider_tests.py

```diff
@@ -72,10 +72,6 @@
         MINIMUM_VERSION: "'1.20.0'",
         "comment": "# NetworKmanager 1.20.0 added support for forgetting profiles",
     },
-    "playbooks/tests_ethtool_coalesce.yml": {
-        MINIMUM_VERSION: "'1.25.1'",
-        "comment": "# NetworkManager 1.25.1 introduced ethtool coalesce support",
```

Comment:
Seems like this needs the new conditional tests support

Reply:
Will change it after that https://github.com/linux-system-roles/network/pull/386 was merged.

Reply:
please just remove this commit from this PR, then.

Reply:
This commit was removed. I think that I may need to add the commit to #386 then.